collect_app/src/androidTest/java/org/odk/collect/android/support/pages/FormHierarchyPage.java

```diff
@@ -22,6 +25,12 @@ public FormHierarchyPage(String formName) {
     @NonNull
     @Override
     public FormHierarchyPage assertOnPage() {
+        // Make sure we wait for loading to finish
+        WaitFor.waitFor((Callable<Void>) () -> {
```

Comment:
This is duplicated from `FormEntryPage`. Good to have here as well as we know form loading can cause flakey interactions with Espresso.